### DISCUSSION TITLE
QSO Detail / Edit dialog

### DISCUSSION THREAD

**Discussion by dl2ki** [foldynl/QLog#96] [General]
After a few days of vacation, I tried out QLog 0.11. The 'QSO Detail / Edit dialog' looks very good!

Thanks for your work on QLog.

**Comment by foldynl** [comment]
Many thanks for your feedback. I hope that the upcoming 0.12 will be the next step forward - redesigned dxc/Bandmap core promises an increased performance. I am testing them under the load of 120k spots per hour and CPU is acceptable.

**Reply by W1DGL-1** [reply]
I also posted this question under "Lookups", but I also like the "zoom-in" function on the edit window map, but why does (when I go to save the lookup info) it say that "yellow areas are invalid"?
I'm doing my lookups via QRZ, so they're not actually 'invalid'.
